.gitignore

```diff
@@ -4,5 +4,6 @@
 pkg/*
 .DS_Store
 .pt_project_id
+.geordi.yml
```

Comment:
If you need to ignore a `.geordi.yml` generated by tests, maybe the test is also writing to the homedir? That's not too good. Please set the setting file paths in geordi/settings.rb depending on `ENV['GEORDI_TESTING']` (set in features/support/env.rb), and have them point to ./tmp/{local_settings,global_settings}.yml in tests.